Sources/Rendering/Core/SphereMapper/example/index.js

```diff
@@ -87,11 +92,50 @@ renderer.addActor(actor);
 renderer.resetCamera();
 renderWindow.render();
 
+const interactor = renderWindow.getInteractor();
+const apiSpecificRenderWindow = interactor.getView();
+const hwSelector = apiSpecificRenderWindow.getSelector();
+hwSelector.setCaptureZValues(true);
+hwSelector.setFieldAssociation(FieldAssociations.FIELD_ASSOCIATION_CELLS);
+
+function eventToWindowXY(event) {
+  // We know we are full screen => window.innerXXX
+  // Otherwise we can use pixel device ratio or else...
+  const { clientX, clientY } = event;
+  const [width, height] = apiSpecificRenderWindow.getSize();
+  const x = Math.round((width * clientX) / window.innerWidth);
+  const y = Math.round(height * (1 - clientY / window.innerHeight)); // Need to flip Y
+  return [x, y];
```

Comment:
can't you use `displayToNormalizedDisplay()` or some other related function?

Reply:
I've used the same function as in the HardwareSelector example
The y coordinates needs to be flipped, I don't know if there is a function which does this kind of things
Using `displayToNormalizedDisplay()` doesn't work as expected, the x coordinate isn't 1 when my mouse is at the right border of the page